src/java/org/apache/cassandra/metrics/DecayingEstimatedHistogramReservoir.java

```diff
@@ -83,6 +83,7 @@
      * The default number of decayingBuckets. Use this bucket count to reduce memory allocation for bucket offsets.
      */
     public static final int DEFAULT_BUCKET_COUNT = 164;
+    public static final int FEWER_BUCKET_COUNT = 127;
```

Comment:
This is a fun naming problem. (Completely non-binding) alternatives could be `LOW_BUCKET_COUNT` or `MICRO_LATENCY_BUCKET_COUNT`. I'm fine with whatever you think sounds best though.

Reply:
OK. Let's do `LOW_BUCKET_COUNT`